### PR TITLE
Files plugin tweaks

### PR DESCRIPTION
**Pull request by avahowell**
this PR makes `uploads` come after `downloads` in the file transfers list, changes styling to use the correct pointer types, and fixes a typo.

### CODE REVIEW

File: plugins/Files/js/components/filetransfers.js [hunk 1]
```diff
@@ -12,8 +12,8 @@ const FileTransfers = ({uploads, downloads, actions}) => {
 			<div className="close-button" onClick={onCloseClick}>
 				<i className="fa fa-times fa-2x"></i>
 			</div>
-			{uploads.size > 0 ? <UploadList uploads={uploads} /> : null}
 			{downloads.size > 0 ? <DownloadList downloads={downloads} onDownloadClick={onDownloadClick} /> : null}
+			{uploads.size > 0 ? <UploadList uploads={uploads} /> : null}
```
Comment (lukechampine) [inline plugins/Files/js/components/filetransfers.js:16]: doesn't `UploadList` need to be passed a `onUploadClick`?

Comment (avahowell) [inline plugins/Files/js/components/filetransfers.js:16]: no, `onUploadClick` gets passed to transfer list's `onTransferClick`, which has an es6 default parameter value.  If you pass undefined, it's the same as passing `() => {}`

**Comment by avahowell** [comment]
dont merge yet, found a bug

**Comment by avahowell** [comment]
Should be ready now

**Pull request merged**